File: 백준/Bronze/1546. 평균/평균.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        double[] arr = new double[in.nextInt()];
        double big = 0.0, sum = 0.0;
        for(int i = 0; i < arr.length; i++){
            arr[i] = in.nextDouble();
            if(big < arr[i])
                big = arr[i];
        }
        for(int i = 0; i <arr.length; i++){
            sum += arr[i]/big*100;
        }
        System.out.print(sum/arr.length);
    }
}

File: 백준/Bronze/1546. 평균/평균.kt
import java.util.Scanner
fun main(){
    val sc = Scanner(System.`in`)
    var arr = DoubleArray(sc.nextInt(), {0.0})
    var big = 0.0; var sum = 0.0
    for(i in 0..(arr.size-1)){
        arr[i] = sc.nextDouble()
        if(big < arr[i])
            big = arr[i]
    }
    for(i in 0..(arr.size-1))
        sum += arr[i]/big*100
    println(sum/arr.size)
}